deploy/cloud-foundry/README.md

```diff
@@ -2,15 +2,42 @@
 
 ## Deployment Steps
 
-The quickest way to install Stratos is to deploy it as a Cloud Foundry application. To do so, clone the `stratos` repository, cd into the newly cloned repository and push to Cloud Foundry. This can be done with:
+The quickest way to install Stratos is to deploy it as a Cloud Foundry application. 
+
+You can do it in two ways:
+
+1. Deploy Stratos using the [`splatform/stratos`](https://hub.docker.com/r/splatform/stratos) docker image 
+
+    This can be done with command `cf push -f manifest-docker.yml`:
+    ```yaml
+    # manifest-docker.yml
+    # https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#docker
+    ---
+    applications:
+    - name: console
+      docker:
+        image: splatform/stratos:latest
```

Comment:
Like the standard cf push instructions we'd like to point the user to a stable docker image. We're just working on creating one at the moment.

Reply:
It's good to know, should I expect `splatform/stratos:stable`?

Reply:
done: https://github.com/cloudfoundry-incubator/stratos/pull/3294/commits/19dd45d564a9926002f35b37e83fc67c1b3a640b